packages/cli/package.json

```diff
@@ -54,7 +54,7 @@
     "@zowe/cics-for-zowe-sdk": "6.0.0-next.202402081454"
   },
   "peerDependencies": {
-    "@zowe/imperative": "^8.0.0-next"
+    "@zowe/imperative": ">8.0.0-next.0 <8.0.0-next.x"
```

Comment:
Should we document somewhere the format we want to use for peer dep version constraints in vNext branches?

At the time that v2 was vNext, we used a different format [here](https://github.com/zowe/zowe-cli-cics-plugin/commit/1638341702d039960945dd9f9ad713be8ded9b63):
`"@zowe/imperative": ">=5.0.0-next.202106221817 <5.0.0"`

Reply:
I think we should. 

I don't know if something changed in newer versions of NPM, but...

This causes the warning to show up at plugin install time due to peerDep version not matching if you have anything other than the specified date for the prerelease 😢